src/Tests/Http/ContentNegotiation/MediaTypeFormatters/FormUrlEncodedMediaTypeFormatterTest.php

```diff
@@ -79,6 +80,7 @@ public function testReadingFromStreamDeserializesStreamContents(): void
     public function testReadingTypeThatCannotBeReadThrowsException(): void
     {
         $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Opulence\Net\Http\ContentNegotiation\MediaTypeFormatters\FormUrlEncodedSerializerMediaTypeFormatter cannot read type string');
```

Comment:
Can you update this to grab the fully-qualified class name dynamically?  That will help a tiny bit if namespaces ever change.